Agents/ForecastingAgent/README.md

```diff
@@ -190,7 +190,7 @@ The following UML diagram provides an overview of how the agent works:
 
 ## 2.2 Usage of Custom Forecasting Models
 
-To use pre-trained/custom models, both a model loading and covariate loading funtion need to be specified in the [model mapping] file, and referenced in the `FC_MODELS` dictionary. Specify any custom loading functions following the example of the `tft_pirmasens_heat_demand` model configuration.
+To use pre-trained/custom models, both a model loading and covariate loading function need to be specified in the [model mapping] file, and referenced in the `FC_MODELS` dictionary. Specify any custom loading functions following the example of the `tft_pirmasens_heat_demand` model configuration. [Building the agent](#31-building-the-agent) again after adding custom models is not necessary since the models will be included as volume when deploying the agent locally.
```

Comment:
This is also the case when not deployed "locally". Hence, I find this a bit confusing. 
As it would now be possible to also just pull the Agent docker image and deploy it, I suggest to clarify a bit more that the content of `fcmodels` needs to be available locally to be mapped according to the bind mount paths